packages/material-ui-lab/src/index.js

```diff
@@ -1,4 +1,16 @@
 /* eslint-disable import/export */
+export { default as AdapterDateFns } from './AdapterDateFns';
```

Comment:
We should probably leave a note that we don't want the adapters here. These import optional peer dependencies which would cause "Module not found" when importing from `@material-ui/lab`.

The re-ordering of `CalendarPickerSkeleton` is fine though.